src/derivatives/atlas.md

```diff
@@ -472,6 +473,141 @@ A guide for using macros can be found at
+template and (*default*) atlas:
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+   "ps13-with-atlases-pipeline": {
+      "tpl-PS13": {
+         "pet": {
+            "tpl-PS13_atlas-Economo1916_desc-nopvc_dseg.nii.gz": "",
+            "tpl-PS13_atlas-Economo1916_desc-pvc_dseg.nii.gz": "",
+            "tpl-PS13_atlas-Economo1916_dseg.json": "",
+            "tpl-PS13_atlas-Economo1916_dseg.tsv": "",
+            "tpl-PS13_atlas-RamonCajal1908_desc-nopvc_dseg.nii.gz": "",
+            "tpl-PS13_atlas-RamonCajal1908_desc-pvc_dseg.nii.gz": "",
+            "tpl-PS13_atlas-RamonCajal1908_dseg.json": "",
+            "tpl-PS13_atlas-RamonCajal1908_dseg.tsv": "",
+            "tpl-PS13_desc-nopvc_dseg.nii.gz": "",
+            "tpl-PS13_desc-pvc_dseg.nii.gz": "",
+            "tpl-PS13_dseg.json": "",
+            "tpl-PS13_dseg.tsv": "",
+            "tpl-PS13_stat-std_desc-fnirtNopvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-fnirtNopvc_mimap.nii.gz": "",
+            "tpl-PS13_stat-std_desc-fnirtPvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-fnirtPvc_mimap.nii.gz": "",
+            "tpl-PS13_stat-std_desc-nopvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-nopvc_mimap.nii.gz": "",
+            "tpl-PS13_stat-std_desc-pvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-pvc_mimap.nii.gz": "",
+         },
+      },
+   }
+})
+}}
+
+where the `atlas-RamonCajal1908` and `atlas-Economo1916` hypothetically define
+two different atlases (please note that, often, atlases are named after
+the first author and indicating a year of a reference communication).
+The original *default* or *implicit* atlas' artifacts such as
+the `tpl-PS13_desc-nopvc_dseg.nii.gz` segmentation,
+which were originally generated with the PS13 template,
+MAY take an [`atlas-<label>`](../glossary.md#atlas-entities) if they
+need to be differentiated from the original template and atlas dataset.
+However, it is RECOMMENDED that these *default* or *implicit* atlases employed
+in the *custom* space they were generated only define
+[`atlas-<label>`](../glossary.md#atlas-entities)
+if it is necessary to disambiguate two or more atlases.
+
+A further example of these template-and-atlas specifications
+is the *Spatially Unbiased Infratentorial Template (SUIT)*
+([Diedrichsen, 2006](https://doi.org/10.1016/j.neuroimage.2006.05.056)):
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+   "suit-pipeline": {
+      "tpl-SUIT": {
+         "anat": {
+            "CHANGES": "",
+            "LICENSE": "",
+            "README.md": "",
+            "dataset_description.json": "",
+            "tpl-SUIT_T1w.nii.gz": "",
```

Comment:
We know that SUIT was originally by Diedrichsen defined in MNIXXX space, but a user might want to have the space entity here again.

Reply:
The template is distributed with transformations to MNI152NLin2009cAsym. I don't have the details very fresh in my mind now, but that suggests it was generated in a space that can be aligned *better* to MNI.

My friend @yasseraleman is using it and indeed, he first aligns SUIT's template and then either use its transforms to move everything back to MNI or bring labels from there.

I mentioned the SUIT example precisely because, despite being *close* to MNI, it's not quite there (hence, deserving it's own space denomination).